src/Controller/Views/CalendarController.php

```diff
@@ -17,7 +17,7 @@ class CalendarController extends AppController
      * @param Request $request
      * @return Response
      *
-     * @Route("/views/calendar/replan/{view}.json", name="views.calendar.replan", defaults={"_format": "json"}, methods={"POST"})
+     * @Route("/views/calendar/replan/{view}.json", name="views.calendar.replan", methods={"POST"})
```

Comment:
_format forces the response mime type I would not remove it without testing it

Reply:
https://symfony.com/doc/current/routing.html#special-parameters

Indeed I will roll back these _format removals (did them by hand on only two places I think).

Reply:
Oh yeah I removed them because it was hard coded in the route itself.
Changing to a minimal diff in my next commit.

However I think the proper solution would be this: 
```suggestion
     * @Route("/views/calendar/replan/{view}.{_format}", name="views.calendar.replan", requirements={"_format"="json"}, methods={"POST"})
```